rustls/src/webpki/client_verifier.rs

```diff
@@ -281,6 +281,8 @@ impl WebPkiClientVerifier {
     /// Construct a new `WebpkiClientVerifier`.
     ///
     /// * `roots` is a list of trust anchors to use for certificate validation.
+    /// * `root_hint_subjects` is a list of distinguished names to use for hinting acceptable
```

Comment:
Squash this into the earlier commit that renamed these and added a bunch of docs?

Reply:
Done :ballot_box_with_check: